storage-api/src/java/org/apache/hadoop/hive/common/ValidTxnWriteIdList.java

```diff
@@ -32,12 +32,18 @@
    */
   public static final String VALID_TABLES_WRITEIDS_KEY = "hive.txn.tables.valid.writeids";
 
+  /**
+   * Key used to store valid write id list for compactor in a
+   * {@link org.apache.hadoop.conf.Configuration} object.
+   */
+  public static final String COMPACTOR_VALID_TABLES_WRITEIDS_KEY = "hive.compactor.txn.tables.valid.writeids";
+
   // Transaction for which the list of tables valid write Ids are populated
   private Long txnId;
 
   // Map of valid write ids list for all the tables read by the current txn
   // Key is full table name string of format <db_name>.<table_name>
-  private Map<String, ValidWriteIdList> tablesValidWriteIdList = new HashMap<>();
+  final private Map<String, ValidWriteIdList> tablesValidWriteIdList = new HashMap<>();
```

Comment:
I think we order these differently.
private final seems more natural - I think that is the expected order

Reply:
Sure